addusers.py

```diff
@@ -201,7 +203,106 @@ def exists_in_keystone(check_thing, keystone_things):
         return None
 
 
-def parse_rows(rows, select_user=None):
+def remove_duplicates(rows, UC, PC, EC):
+    """Deletes all duplicate rows.
+    user_id_column: UC
+    new_project_column: PC
+    existing_project_column: EC
+    """
+    if len(rows) <= 1:
+        return rows
+    else:
+        output = []
+        for i in range(len(rows)):
+            flag = True
+            for j in rows[:i]:
+                if rows[i][UC] == j[UC] and\
+                   rows[i][PC] == j[PC] and\
+                   rows[i][EC] == j[EC]:
+                    flag = False
+            if flag:
+                output.append(rows[i])
+    return output
+
+
+def create_new_users_rows(rows):
+    output = []
+    for entry in rows:
+        if entry[7] == 'No':
+            user = [entry[4], entry[5], entry[6], entry[8],
+                    entry[9], entry[10], entry[11], entry[12],
+                    entry[13], entry[14], entry[15], entry[16],
+                    entry[18], entry[19], entry[20], entry[21]]
+            output.append(user)
+    return output
+
+
+def create_new_projects_rows(rows):
+    output = []
+    for entry in rows:
+        if entry[15] != "":
+            project = [entry[4], entry[5], entry[6], entry[8],
+                       entry[9], entry[10], entry[11], entry[12],
+                       entry[13], entry[16], entry[17]]
+            output.append(project)
+    return output
+
+
+def manual_testing(header):
```

Comment:
So, this is solving a completely different problem than the original PR.  In general, each PR should deal with a single topic to make things easy for reviewers.

Can you please submit a different PR for each bug you are fixing?